.github/workflows/test_and_lint.yaml

```diff
@@ -1,4 +1,4 @@
-name: Test and lint
+name: Deploy Package
```

Comment:
What is this doing now? are we planning to run tests later?

Reply:
Copy-paste error - will fix!

Reply:
cool but we still want to run pytest etc. don't we?